packages/ember/tests/routing/basic_test.js

```diff
@@ -321,90 +321,6 @@ QUnit.test('Model passed via renderTemplate model is set as controller\'s model'
   equal(jQuery('p:contains(emberjs)', '#qunit-fixture').length, 1, 'Passed model was set as controllers model');
 });
 
-test('Renders correct view with slash notation', function() {
-  setTemplate('home/page', compile('<p>{{view.name}}</p>'));
-
-  Router.map(function() {
-    this.route('home', { path: '/' });
-  });
-
-  App.HomeRoute = Route.extend({
-    renderTemplate() {
-      this.render('home/page');
-    }
-  });
-
-  App.HomePageView = EmberView.extend({
-    name: 'Home/Page'
-  });
-
-  bootApplication();
-
-  equal(jQuery('p:contains(Home/Page)', '#qunit-fixture').length, 1, 'The homepage template was rendered');
-});
-
-test('Renders the view given in the view option', function() {
```

Comment:
I don't know if this can be removed or not. This is talking about things registered as `view:${someView}` in the registry. If it's no longer supported we can't just delete the test the feature needs to be removed as well. This means you have to clean up all of this code. https://github.com/emberjs/ember.js/blob/master/packages/ember-routing/lib/system/route.js#L1962-L1978